### ISSUE TITLE
Unifi Doorbell Motion & Bell Action Disconnection on Virtual Machine OS

### ISSUE THREAD

**Issue by houshasen** [briis/unifiprotect#219]
### Issue

Unifi Doorbell Motion and Bell Ring trigger stops working after few hours while Home Assistant OS is running on the Virtual Machine. The same setup, but Home Assistant OS on Rasberry Pi continues to work.

Video stream continues to work so it is not disconnection of the device. Also, rebooting of the Home Assistant OS on the Virtual Machine temporary fix the issue i.e. Not issue with UniFi Protect or Doorbell side.

### Setups

Home Assistant OS Version: Core-2021.4.0
UniFI Protect Integration Version: 0.8.5
UniFi Protect version: 1.17.4

### Potential Relevant Information

I surmise this is the same/similar issue to: #https://github.com/briis/unifiprotect/issues/197

I believe this is a similar issue raised on Homebridge community. Here is a link that developer claims to have got it fixed but not yet released his update.

https://github.com/hjdhjd/homebridge-unifi-protect/issues/286#issuecomment-776032418

**Comment by briis** [comment]
Same issue as #218 it seems. I will look into this as soon as I can

**Comment by houshasen** [comment]
Here is a little update. Couple days after I posted this, my UDM-Pro got updated to 1.9.3 (official release). Although UniFi Protect Version remains 1.17.4, Motion & Bell Actions seem to stay working since then. Before I got those disconnected within a few hours and never lasted a day, so I wonder this is just a coincident and all I had to do was UDM Pro reboot (happened automatically as part of upgrade) or 1.9.2 UDM-Pro firmware itself was faulty.

In any event, I wanted to post this follow up. So in case developers working and may not be able to replicate the issue anymore.

Thank you
